### PR TITLE
Fix the stupids

### LINKED ISSUE #11: Stupids
- change package name to to "com.whereuat.whereuat". Or better yet, why not just "com.whereuat"? Do we have the technology? Either way, "com.whereuat.whereu" looks dirty.
- Change `mVerfiyCode` to `mVerificationEdit` in `LoginActivity`
- `public static final String ACCOUNT_NEW_BROADCAST = "com.whereuat.whereu.ACCOUNT_NEW_BROADCAST";` can be one line in `Constants`
- change Internet permission line to have a space before `/>`

### PR DESCRIPTION
**Pull request by kangp3**
Closes #11

### CODE REVIEW

File: app/src/main/java/xyz/whereuat/whereuat/Constants.java [hunk 1]
```diff
@@ -17,9 +19,7 @@
 
     public static final String ACCOUNT_REQUEST_BROADCAST =
             "com.whereuat.whereu.ACCOUNT_REQUEST_BROADCAST";
-    public static final String ACCOUNT_NEW_BROADCAST =
-            "com.whereuat.whereu.ACCOUNT_NEW_BROADCAST";
-    public static final String TOKEN_BROADCAST =
-            "com.whereuat.whereu.TOKEN_BROADCAST";
+    public static final String ACCOUNT_NEW_BROADCAST = "com.whereuat.whereu.ACCOUNT_NEW_BROADCAST";
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/Constants.java]: Please change these to use the new package name

File: app/src/main/java/xyz/whereuat/whereuat/AutoShareStar.java [hunk 2]
```diff
@@ -22,12 +22,12 @@
 
     public AutoShareStar(Context context, AttributeSet attrs) {
         super(context, attrs);
-        TypedArray ta = context.obtainStyledAttributes(attrs, R.styleable.AutoShareStar);
+        TypedArray ta = context.obtainStyledAttributes(attrs, xyz.whereuat.whereuat.R.styleable.AutoShareStar);
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/AutoShareStar.java]: Why do we need to prefix this with the package name? Are you sure we need to do this?

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 3]
```diff
@@ -118,23 +120,28 @@ public void onReceive(Context context, Intent intent) {
...
         }
     }
 
     @Override
-    public void onActivityResult(int reqCode, int resultCode, Intent data) {
-        super.onActivityResult(reqCode, resultCode, data);
-        ContactRetriever con = new ContactRetriever(reqCode,resultCode, data, this.getApplicationContext());
+    public void onActivityResult(int req_code, int result_code, Intent data) {
+        super.onActivityResult(req_code, result_code, data);
+        ContactRetriever con = new ContactRetriever(req_code,result_code, data, this);
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: Space between req_code and result_code :)

File: app/src/main/java/xyz/whereuat/whereuat/db/WhereuatContract.java [hunk 4]
```diff
@@ -1,16 +1,10 @@
-package com.whereuat.whereu.db;
+package xyz.whereuat.whereuat.db;
 
-import android.content.Context;
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/db/WhereuatContract.java]: Did we just have a ton of unused imports?

File: app/src/main/java/xyz/whereuat/whereuat/Constants.java (hunk 1 above)
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/Constants.java]: Done

File: app/src/main/java/xyz/whereuat/whereuat/AutoShareStar.java (hunk 2 above)
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/AutoShareStar.java]: Done

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java (hunk 3 above)
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: Done

**Pull request merged**